### ISSUE TITLE
Simplify the Bug Report creation flow

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#21611]
### What happened?

A bit too many fields in this form.

### What should happen?

Proposed steps:

1. Remove Matomo version select box
2. instead, because people wouldn't know about `Matomo minor or patch Version`, then we can ask them the full version number in 1 field only? 
3. Make PHP version optional
4. Make OS optional
5. Make Computer OS optional

### How can this be reproduced?

Create a new bug, and be impressed by the complexity. 

### Matomo major version

Matomo 3

### Matomo minor or patch Version

???

### PHP version

???

### Server operating system

???

### Computer operating system

???

### Validations

- [X] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [X] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.

**Comment by michalkleiner** [comment]
We have updated the bug report template recently in agreement with QA in order to get better bug reports with more data for easier issue replication. We can discuss if and how we can possibly simplify it even more while still getting quality input. We can perhaps socialise this also on the forum for more community feedback if we think it's a good idea.

Removed the Bug tag as this is not an issue with Matomo itself. Over to product and QA for discussion/prioritisation.
